src/utils/api.js

```diff
@@ -154,17 +154,17 @@ class Api {
 
 	get = (type, id, include = '') => {
 		if (!id) { return Promise.reject(new Error('id is missing')); }
-		return this._sendAuthenticatedRequest('GET', `/api/v2/${type}/${id}?${include ? `include=${include}` : ''}`);
+		return this._sendAuthenticatedRequest('GET', `/api/v2.1.0/${type}/${id}?${include ? `include=${include}` : ''}`);
```

Comment:
@christophertino Should we pull the API version in from a global that we set by calling account at startup so that we can upgrade the version without having to do an extension release?

Reply:
Leaving as is